monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -4685,8 +4708,11 @@ private[eval] abstract class TaskInstancesLevel0 extends TaskParallelNewtype {
     */
   implicit def catsEffect(
     implicit s: Scheduler,
-    opts: Task.Options = Task.defaultOptions): CatsConcurrentEffectForTask =
-    new CatsConcurrentEffectForTask
+    opts: Task.Options = Task.defaultOptions): CatsConcurrentEffectForTask = {
+
+    val opts2 = opts.withSchedulerFeatures
```

Comment:
It might be redundant because running `Task` (e.g. in `runAsync`) checks it anyway

Reply:
Ah, good point.

Reply:
Fixed.